src/Sarif.WorkItems/SarifWorkItemFiler.cs

```diff
@@ -336,7 +336,9 @@ public SarifWorkItemModel FileWorkItemInternal(SarifLog sarifLog, SarifWorkItemC
 
                     if (string.IsNullOrWhiteSpace(sarifWorkItemModel.Title) || string.IsNullOrWhiteSpace(sarifWorkItemModel.BodyOrDescription))
                     {
-                        this.Logger.LogWarning("Attempt to call work item client with invalid work item values.", logId);
+                        this.Logger.LogWarning("Attempt to call work item client with invalid work item values. Title: '{sarifWorkItemModel.Title}', BodyOrDescription:{sarifWorkItemModel.BodyOrDescription}",
```

Comment:
This is something I was seeing as an error, but is probably a warning for other people.  It's a pretty obvious "just fix it" case.

I want to say I'm going to be methodical about rest of the warning work.  I'm only frontloading these two issues on an ad-hoc basis because a) they're causing pain locally, and b) the resolutions are obvious and should be noncontroversial.